docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -13,6 +13,9 @@ New Features
    processes are running. This will prevent the user from corrupting their Pulp installation by
    migrating with running workers. This works in both standalone and clustered installations. Pulp
    may wait up to 92 seconds to determine if workers are running.
+ * pulp_workers, pulp_resource_manager, and pulp_celerybeat record their own hearbeats to the
```

Comment:
I think the sentence order should be reversed. The most valuable statement for users should come first. How it works is a secondary concern.